src/FindApprenticeshipTraining/SFA.DAS.FindApprenticeshipTraining.UnitTests/Application/Courses/Queries/GetCourseByLarsCode/WhenGettingCourseByLarsCode.cs

```diff
@@ -0,0 +1,194 @@
+            x.GetWithResponseCode<GetCourseTrainingProvidersCountResponse>(
+                It.IsAny<GetCourseTrainingProvidersCountRequest>()
+            ), Times.Once);
+    }
+
+    [Test]
+    public async Task Handle_Returns_Default_When_No_Providers()
+    {
+        var query = new GetCourseByLarsCodeQuery { LarsCode = 456 };
+
+        var standardDetailResponse = new StandardDetailResponse
+        {
+            ApprenticeshipFunding = new List<ApprenticeshipFunding>
+            {
+                new ApprenticeshipFunding { MaxEmployerLevyCap = 7000, Duration = 24 }
+            },
+            Ksbs = new List<KsbResponse> 
+            { 
+                new KsbResponse 
+                { 
+                    Type = GetCourseByLarsCodeQueryHandler.KsbsSkillsType,
+                    Description = "Skill B" 
+                } 
+            }
+        };
+
+        var emptyCourseProvidersResponse = new GetCourseTrainingProvidersCountResponse
+        {
+            Courses = new List<CourseTrainingProviderCountModel>()
+        };
+
+        _coursesApiClientMock
+            .Setup(x => x.GetWithResponseCode<StandardDetailResponse>(
+                It.Is<GetStandardDetailsByIdRequest>(a => 
+                    a.Id.Equals(query.LarsCode.ToString())
+                )
+            ))
+            .ReturnsAsync(
+                new ApiResponse<StandardDetailResponse>(
+                    standardDetailResponse, 
+                    HttpStatusCode.OK, 
+                    string.Empty
+                )
+            );
+
+        _roatpCourseManagementApiClientMock
+            .Setup(x => 
+                x.GetWithResponseCode<GetCourseTrainingProvidersCountResponse>(
+                    It.Is<GetCourseTrainingProvidersCountRequest>(a => 
+                        a.LarsCodes.SequenceEqual(new int[1] { query.LarsCode }) &&
+                        a.Distance.Equals(query.Distance) &&
+                        a.Latitude.Equals(query.Lat) &&
+                        a.Longitude.Equals(query.Lon)
+                    )
+                )
+            )
+            .ReturnsAsync(
+                new ApiResponse<GetCourseTrainingProvidersCountResponse>(
+                    emptyCourseProvidersResponse, 
+                    HttpStatusCode.OK, 
+                    string.Empty
+                )
+            );
+
+        var sut = await _handler.Handle(query, CancellationToken.None);
+
+        Assert.Multiple(() =>
+        {
+            Assert.That(sut, Is.Not.Null);
+            Assert.That(sut.MaxFunding, Is.EqualTo(7000));
+            Assert.That(sut.TypicalDuration, Is.EqualTo(24));
+            Assert.That(sut.ProvidersCount, Is.EqualTo(0));
+            Assert.That(sut.TotalProvidersCount, Is.EqualTo(0));
+            Assert.That(sut.Skills[0], Is.EqualTo("Skill B"));
+            Assert.That(sut.Knowledge, Is.Empty);
+            Assert.That(sut.Behaviours, Is.Empty);
```

Comment:
This deserves a test on its own, it is hidden at the moment in the no provider test.